### PR TITLE
Examples refactoring no 1,2,3 (after issue #12)

### PR DESCRIPTION
**Pull request by Samox1**
Examples refactoring - no 1, 2, 3 - using changes made in commits in the branch for Issue #12 
Every comment will be helpful

### CODE REVIEW

File: example_1_basic_read/main.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import os
+
+from speakleash import Speakleash
+
+
+if __name__ == '__main__':
+
+    base_dir = os.path.join(".")
+    replicate_to = os.path.join(base_dir, "datasets")
+    print("Replicate to:", replicate_to)
```
Comment (IgorTest19) [inline example_1_basic_read/main.py]: More common thing is to use f'string formatting like:
print(f"Replicate to: {replicate_to}")

File: example_2_inventory_check/main.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        sources = manifest.get("sources", [])
+        if len(sources) == 1:
+            licence = (sources[0].get("license", colored("n/a", "red"))
+                        .replace("/n", "").replace(";", ",").strip())
+            license_url = sources[0].get("license_url", "n/a")
+        else:
+            licence = "multiple"
+            license_url = "n/a"
+
+        # print("{0};{1} MB;{2};{3};{4};{5}".format(colored(d.name,'green'), size_mb,
```
Comment (IgorTest19) [inline example_2_inventory_check/main.py]: Commented out code can be deleted in this case.

**Review by IgorTest19 — CHANGES_REQUESTED** [review]

File: example_1_basic_read/main.py (hunk 1 above)
Comment (Samox1) [reply example_1_basic_read/main.py]: Missed that - the same change will be in example no. 3 (quality_metrics)

File: example_2_inventory_check/main.py (hunk 2 above)
Comment (Samox1) [reply example_2_inventory_check/main.py]: Removed - replaced with a comment explaining another piece of code

**Review by IgorTest19 — APPROVED** [review]

**Pull request merged**